test/browser/lifecycles/shouldComponentUpdate.test.js

```diff
@@ -707,4 +707,24 @@ describe('Lifecycle methods', () => {
 			expect(scratch.textContent).to.equal('foo');
 		});
 	});
+
+	it('should correctly render when sCU component has null children', () => {
```

Comment:
This test was in this branch from a while ago. Not sure what it was trying to do but figured it can't hurt 🤷‍♂